backend/sftp/sftp.go

```diff
@@ -86,6 +87,12 @@ in the new OpenSSH format can't be used.`,
 			Help: `Optional path to public key file.
 
 Set this if you have a signed certificate you want to use for authentication.` + env.ShellExpandHelp,
+		}, {
+			Name: "knownhosts_file",
```

Comment:
The file is called `~/.ssh/known_hosts` so we should probably call the config option `known_hosts_file`

Reply:
But the OpenSSH configuration option is `UserKnownHostsFile` and `GlobalKnownHostsFile`, so it's all so confusing :-)   I can change the name if you think it's worth it.

Reply:
OK, change made.